daemon/graphdriver/aufs/aufs.go

```diff
@@ -229,12 +235,16 @@ func (a *Driver) Remove(id string) error {
 	a.Lock()
 	defer a.Unlock()
 
-	if a.active[id] != 0 {
-		logrus.Errorf("Removing active id %s", id)
```

Comment:
If someone wants to double check this it would help.  This looked like a bug to me that aufs would still remove and unmount a container if the reference count was `> 0`